city_scrapers/spiders/il_capital_development.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlCapitalDevelopmentSpider(CityScrapersSpider):
+    name = "il_capital_development"
+    agency = "Illinois Capital Development Board"
+    timezone = "America/Chicago"
+    start_urls = ["https://www2.illinois.gov/cdb/about/boardmeetings"]
+    location = {
+        "name": "James R. Thompson Center",
+        "address": "100 West Randolph Street, 14th Floor, Chicago, IL 60601"
+    }
+
+    def parse(self, response):
+        """
+        """
+        for item in response.xpath("//tbody/tr"):
+            self._validate_location(item)
+            meeting = Meeting(
+                title="CDB Board Meeting",
+                description=self._parse_description(item),
+                classification=BOARD,
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self.location,
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
```

Comment:
We should pass the text that's currently getting pulled in `description` to the `text` kwarg here which will check for any mention of cancellations.

The `_get_status` function checks for any mention of words like cancel and reschedule, and since the notes about "Time Changed" seem to reflect the current details of the row it looks like `_get_status` should cover it